libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationView.java

```diff
@@ -72,7 +74,7 @@
  * @since 0.7.0
  */
 public class NavigationView extends CoordinatorLayout implements LifecycleOwner, OnMapReadyCallback,
-  NavigationContract.View {
+    NavigationContract.View {
```

Comment:
`Checkstyle` fails if I add this change

Reply:
Ohh I believe it's because of the continuation indent https://github.com/mapbox/mapbox-navigation-android/blob/3561a68cc4999bafdfc7168a79c53315177b71e7/config/mapbox-java-codestyle.xml#L81 probably we probably didn't have that rule in the past. In any case, it's weird that `Checkstyle` fails as it's not failing in the legacy 🤔 Actually, that checkstyle rule is disabled https://github.com/mapbox/mapbox-navigation-android/blob/3561a68cc4999bafdfc7168a79c53315177b71e7/config/checkstyle/checkstyle.xml#L430-L432